airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -194,7 +194,8 @@ def run_query(
             write_disposition = 'WRITE_EMPTY',
             allow_large_results=False,
             udf_config = False,
-            use_legacy_sql=True):
+            use_legacy_sql=True,
+            maximum_billing_tier=1):
```

Comment:
Was thinking more about this. I think rather than defaulting to `1`, we should default to None or False, and only set the billing tier if it's actually set. Reason for this is the default maximum billing tier in BQ is not actually `1`. See [here](https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.query.maximumBillingTier). It is project specific. To keep this behavior, if the hook/operator are called without explicitly setting the billing tier, I think it shouldn't be set.